SConstruct

```diff
@@ -10,16 +10,16 @@ import socket
 
 
 #----------------------------------------------------------------------
-# Temporary setting for using new Output implementation
+# Temporary setting for using new Refresh implementation
 #----------------------------------------------------------------------
 
-new_output = 0
+new_refresh = 1
 
 #----------------------------------------------------------------------
-# Temporary setting for using new PPM routines from enzo-dev
+# Temporary setting for using new Output implementation
 #----------------------------------------------------------------------
 
-new_ppm = 1
```

Comment:
Why is this parameter getting removed? Or are the old PPM routines getting stripped, making this unnecessary?

Reply:
@aemerick I think I did a quick grep on *pp files and nothing came up, so I assumed this flag was obsolete; but I see that's because NEW_PPM is confined to *.F files.

@anyone Still, is there any reason to keep the old ppm_de.F, [xyz]euler_sweep.F code in Enzo-E?

Reply:
If it is unused, I see no objection to removing it.

Reply:
Ah ok I see. I vote stripping out all of the old PPM code and leave just the “new”/enzo-dev PPM code, getting rid of this flag in the process. I don’t see any reason to keep around the old versions. 

Happy to submit this PR.